dataland-frontend/tests/e2e/utils/GeneralApiUtils.ts

```diff
@@ -132,7 +132,7 @@ export function uploadCompanyViaApiAndEuTaxonomyDataViaForm<T>(
   frameworkDataType: DataTypeEnum,
   companyInformation: CompanyInformation,
   testData: T,
-  formFill: (data: T) => void,
+  formFill: (data: T, storedCompany: StoredCompany) => void,
```

Comment:
I checked the code and I could not find a usage of `uploadCompanyViaApiAndEuTaxonomyDataViaForm()` where the `formFill()` actually uses the `storedCompany`.
So this change might be revertable, right?

Reply:
👍